app/workers/subject_workflow_status_count_worker.rb

```diff
@@ -0,0 +1,21 @@
+class SubjectWorkflowStatusCountWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :really_high
+
+  sidekiq_options congestion: {
+    interval: ENV.fetch("sws_count_worker_interval", 30),
```

Comment:
Indeed it would, we'd have to flesh out the model for it but I agree. You want me to rip the ENV configs out? They may not get used, due the cost of rebuilding the ami's...